common/src/state/chats.rs

```diff
@@ -41,10 +39,10 @@ pub struct Chat {
 // warning: Chats implements Serialize
 #[derive(Clone, Debug, Default, Deserialize)]
 pub struct Chats {
-    #[serde(default)]
+    #[serde(skip)]
     pub initialized: bool,
     // All active chats from warp.
-    #[serde(default)]
+    #[serde(skip)]
```

Comment:
we would want to cache these in state so we don't have to call warp every time I would think, doesn't this break that flow?